src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/Symbols.vb

```diff
@@ -950,9 +950,6 @@ Namespace Microsoft.VisualBasic.CompilerServices
                 Return result
             End Function
 
-            <UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
```

Comment:
This suppression was wrong, all that was needed was to annotate `GetClassConstraint` above.